### PR TITLE
Added VIA support for Drop CTRL

### PR DESCRIPTION
**Pull request by InonL**
## Description

Added a VIA keymap based on the default one with 4 layers (two additional blank layers added for VIA). 

Just like the Drop ALT, It is possible to use VIA with the CTRL - I have tested it with my keyboard and it works.
The .json file for VIA will be added to the VIA repository. 
Exactly like the ALT, RGB customization using VIA does not work but the regular QMK RGB does.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/massdrop/ctrl/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum ctrl_keycodes {
+    U_T_AUTO = SAFE_RANGE, //USB Extra Port Toggle Auto Detect / Always Active
```
Comment (drashna) [inline keyboards/massdrop/ctrl/keymaps/via/keymap.c:20]: If you use the "USER00-USER15" keycodes here, you can assign custom keycodes in the via json, so they are still accessible from VIA. 
```suggestion
    U_T_AUTO = USER00, //USB Extra Port Toggle Auto Detect / Always Active
```

Comment (InonL) [reply keyboards/massdrop/ctrl/keymaps/via/keymap.c]: I wasn't aware of that, thanks!
I've commited the suggested change.

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Comment by rappo** [comment]
Hi @InonL, are you still planning to add the json to the VIA repo, or did I miss it somehow? thanks!

**Comment by InonL** [comment]
> Hi @InonL, are you still planning to add the json to the VIA repo, or did I miss it somehow? thanks!

Yes, linked below. The matrix layout in this board is pretty weird so mapping it took a lot of trial and error and time.

**Pull request merged**